testing/vulnerable_app/util.py

```diff
@@ -8,3 +11,18 @@ def get_name(name):
     :type name: str
     """
     return name.split('.')[-1]
+
+
+class ListConverter(BaseConverter):
+
+    def __init__(self, url_map):
+        super(ListConverter, self).__init__(url_map)
+        self.delimiter = '%2C'
```

Comment:
Probably better to change to the following segment, for self-documenting code.

```
from urllib.parse import quote_plus
self.delimiter = quote_plus(',')
```

but like, with the import at a global level, of course.

Reply:
Done!